### DISCUSSION TITLE
ChakraProvider breaks with the error TypeError: breakpoints.map is not a function in NextJs.

### DISCUSSION THREAD

**Discussion by ChinmayGopal931** [chakra-ui/chakra-ui#6277] [General]
I don't see any information about this only but it seems to be that the the theme is somehow being broken. I traced it back to this line in the node_modules/@styled-system/css/dist/index.esm.js   
```
    var breakpoints = get(theme, 'breakpoints', defaultBreakpoints);
    
    var mediaQueries = [null].concat(breakpoints.map(function (n) {
      return "@media screen and (min-width: " + n + ")";
    }));
```
I think somehow theme is getting the wrong data. I am really not sure what's happening. Can someone point me in the right direction .

![image](https://user-images.githubusercontent.com/47234753/177435652-ec032443-e75c-44c8-8ba9-199fc23a6d65.png)

**Comment by MSAJAL07** [comment]
facing the same issue. any update?
